.github/workflows/build-frontend-image.yaml

```
@@ -101,4 +118,13 @@ jobs:
           aws_default_region: ${{ secrets.AWS_DEFAULT_REGION }}
           azure_acr_hostname: ${{ secrets.AZURE_ACR_HOSTNAME }}
           azure_acr_user: ${{ secrets.AZURE_ACR_USER }}
-          azure_acr_password: ${{ secrets.AZURE_ACR_PASSWORD }}
\ No newline at end of file
+          azure_acr_password: ${{ secrets.AZURE_ACR_PASSWORD }}
```

actionlint (1.7.10)

input "azure_acr_password" is not defined in action "Docker Tag and Push" defined at "./.github/actions/docker-tag-push". available inputs are "aws_account_id", "aws_default_region", "aws_push", "azure_acr_hostname", "azure_push", "conditional_tag", "local_image", "push_tags"

(action)